website/docs/r/monitor_cloud_account.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "Sysdig Monitor"
+layout: "sysdig"
+page_title: "Sysdig: sysdig_monitor_cloud_account"
+description: |- 
+  Creates a Sysdig Monitor Cloud Account 
+---
+
+# Resource: sysdig_monitor_cloud_account
+
+Creates a Sysdig Monitor Cloud Account for monitoring cloud resources.
+
+-> **Note:** Sysdig Terraform Provider is under rapid development at this point. If you experience any issue or discrepancy while using it, please make sure you have the latest version. If the issue persists, or you have a Feature Request to support an additional set of resources, please open a [new issue](https://github.com/sysdiglabs/terraform-provider-sysdig/issues/new) in the GitHub repository.
+
+## Example Usage
+
+```terraform
+resource "sysdig_monitor_cloud_account" "sample" {
+  cloud_provider = "GCP"
+  integration_type = "API"
+  account_id = "A60E650B-B24F-4934-867A-45D5F0DB814E"
```

Comment:
not sure about the `account_id` format.
probably it's just the example sample, but on gcp account_ids are something like 'org-chid-project'